--- charts/vector.orig/templates/_pod.tpl
+++ charts/vector/templates/_pod.tpl
@@ -161,6 +161,9 @@
 {{- with .Values.extraVolumeMounts }}
 {{- toYaml . | nindent 6 }}
 {{- end }}
+{{- if (index .Values "geoip" | default dict).enabled }}
+{{- list (dict "name" "geoip-db" "mountPath" (dir .Values.geoip.database) "readOnly" true) | toYaml | nindent 6 }}
+{{- end }}
 {{- if .Values.extraContainers }}
 {{- tpl (toYaml .Values.extraContainers) . | nindent 2 }}
 {{- end }}
@@ -218,5 +221,8 @@
 {{- end }}
 {{- with .Values.extraVolumes }}
 {{- toYaml . | nindent 2 }}
+{{- end }}
+{{- if (index .Values "geoip" | default dict).enabled }}
+{{- list (dict "name" "geoip-db" "image" (dict "reference" .Values.geoip.image "pullPolicy" "IfNotPresent")) | toYaml | nindent 2 }}
 {{- end }}
 {{- end }}
