# https://github.com/openebs/monitoring/issues/98
--- vendor/github.com/openebs/monitoring/jsonnet/openebs-mixin/rules/prometheus-rules.libsonnet.orig	2022-04-20 11:16:33.096498884 +0200
+++ vendor/github.com/openebs/monitoring/jsonnet/openebs-mixin/rules/prometheus-rules.libsonnet	2022-04-20 11:16:49.831051800 +0200
@@ -23,6 +23,6 @@
   local prometheusRules = self,
   _config+:: param,
   prometheusRules+::
-    cstor(prometheusRules._config).prometheusRules.cStor + jiva(prometheusRules._config).prometheusRules.jiva + lvmLocalPV(prometheusRules._config).prometheusRules.lvmlocalpv + volume(prometheusRules._config).prometheusRules.volume
+    cstor(prometheusRules._config).prometheusRules.cStor + jiva(prometheusRules._config).prometheusRules.jiva + lvmLocalPV(prometheusRules._config).prometheusRules.lvmLocalPV + volume(prometheusRules._config).prometheusRules.volume
     + npd(prometheusRules._config).prometheusRules.npd,
 }
