diff -tubr charts/kube-prometheus-stack/charts/grafana/templates/_pod.tpl charts/kube-prometheus-stack.zdt/charts/grafana/templates/_pod.tpl
--- charts/kube-prometheus-stack/charts/grafana/templates/_pod.tpl	2021-06-26 19:54:57.000000000 +0200
+++ charts/kube-prometheus-stack.zdt/charts/grafana/templates/_pod.tpl	2021-07-01 13:31:20.750609396 +0200
@@ -198,11 +198,16 @@
       - name: FOLDER_ANNOTATION
         value: "{{ .Values.sidecar.dashboards.folderAnnotation }}"
       {{- end }}
+      - name: SCRIPT
+        value: /opt/script.sh
     resources:
 {{ toYaml .Values.sidecar.resources | indent 6 }}
     volumeMounts:
       - name: sc-dashboard-volume
         mountPath: {{ .Values.sidecar.dashboards.folder | quote }}
+      - name: script-volume
+        mountPath: /opt/script.sh
+        subPath: script.sh
 {{- end}}
   - name: {{ .Chart.Name }}
     {{- if .Values.image.sha }}
